content/router.xql

```diff
@@ -326,7 +326,8 @@ declare function router:body ($request as map(*)) {
             :)
             case "xml" return 
                 let $data := request:get-data()
-                let $_test := parse-xml($data)
+                let $_seq-type := $data => util:get-sequence-type()
```

Comment:
$_seq-type is used only once so the call to `util:get-sequence-type($data)` can also be inside of the if-statement.